server/checks/Approval.js

```diff
@@ -6,11 +6,24 @@ import * as EVENTS from '../model/GithubEvents'
 import * as AUDIT_EVENTS from '../service/audit/AuditEventTypes'
 import * as _ from 'lodash'
 
+const IGNORE_LOGIN_RE =[/-robot^/]
```

Comment:
I just realized, that the regex should be `/-robot$/`. Can you maybe add a small unit test?